shotover-proxy/src/message/mod.rs

```diff
@@ -300,7 +300,7 @@ impl Message {
             Metadata::Cassandra(metadata) => {
                 let body = CassandraOperation::Error(ErrorBody {
                     error_code: 0x1001,
-                    message: "".into(),
+                    message: "Server overloaded".into(),
```

Comment:
Is this related to this PR?

Reply:
Yes. Now that we are testing the error messages generically I added this to make that possible. Plus it makes sense to include this message so might as well.